attachment_utils.py

```diff
@@ -197,3 +197,21 @@ def get_is_botany_collection_object_redacted(self, collection_object_id):
                 if val is True or val == 1 or val == b'\x01':
                     return True
         return False
+
+    def get_is_taxon_id_redacted(self, taxon_id):
```

Comment:
Per our discoveries about redaction being collection specific, this probably belongs above the utils level - something botany specific.